app/src/main/res/layout/top_site_item.xml

```diff
@@ -12,14 +12,18 @@
     android:layout_marginBottom="@dimen/top_sites_item_margin_bottom"
     android:orientation="vertical">
 
-    <ImageView
-        android:id="@+id/favicon_image"
-        style="@style/TopSite.Favicon"
-        android:layout_gravity="center"
-        android:importantForAccessibility="no"
+    <com.google.android.material.card.MaterialCardView
```

Comment:
@mcarare I am curious about your thinking here. I see switching to using `MaterialCardView` as a good thing, but I am wondering how you approached the issue and reasoning for switching to `MaterialCardView`.

Reply:
First of all, I wanted to use a `ShapeableImageView` because it comes with simple management of `ImageView` shape. Using a background XML for that would be prone to errors so I switched to using a `CardView` to provide the background because it makes it simpler to add elevation, background color without having to use a separate XML resource. And, at last, the reason for using a `MaterialCardView` instead of a `widget.CardView` was that, besides the material design implementation, it offers properties to change the border color and stroke.

Reply:
That makes a lot of sense. Thanks!